fcrepo-message-consumer-core/src/main/java/org/fcrepo/indexer/NamedFieldsRetriever.java

```diff
@@ -86,37 +91,55 @@ public NamedFields get() {
         try {
             final Model rdf = rdfr.get();
             if (!rdf.contains(createResource(uri.toString()), INDEXING_TRANSFORM_PREDICATE)) {
-                LOGGER.info(
-                        "Found no property locating LDPath transform for: {}, will not retrieve transformed content.",
-                        uri);
-                throw new AbsentTransformPropertyException(uri.toString());
+                LOGGER.info("Looking up property locating LDPath transform for: {}", uri);
+                // make an initial HEAD request and check Link headers for descriptions located elsewhere
+                final HttpHead headRequest = new HttpHead(uri);
+                final HttpResponse headResponse = httpClient.execute(headRequest);
+                URI descriptionURI = null;
+                final Header[] links = headResponse.getHeaders("Link");
+                if ( links != null ) {
+                    for ( Header h : headResponse.getHeaders("Link") ) {
+                        final Link link = Link.valueOf(h.getValue());
+                        if (link.getRel().equals("describedby")) {
+                            descriptionURI = link.getUri();
+                            LOGGER.debug("Using URI from Link header: {}", descriptionURI);
+                        }
+                    }
+                }
+                if (descriptionURI == null) {
+                    throw new AbsentTransformPropertyException("Property lookup failed");
```

Comment:
It will help debugging if the `AbsentTransformPropertyException` also includes the URI of the target resource.